### ISSUE TITLE
docker build fails with non zero code: 2

### ISSUE THREAD

**Issue by BramVan-Oosterhout** [Dieken/foswiki-docker#1]
When the `mirror` variable is not set, the command: ` sudo docker build . -t foswiki` fails with message:

```
/bin/sh: 1: mirror: parameter not set
The command '/bin/sh -c set -eux;     [ -z "$mirror" ] || sed -i -E "s|http(s?)://deb.debian.org|$mirror|" /etc/apt/sources.list' returned a non-zero code: 2
```
It is probably forced by the `set -eu`. Not knowing the larger purpose of this command, I will leave the solution to you.

Kind regards, Bram

**Comment by Dieken** [comment]
The `mirror` build arg is to set mirror site for Debian repository, just push a fix to explicitly set it to empty.

**Comment by BramVan-Oosterhout** [comment]
Thanks Yubao,

Now: perl docker-install.pl dieken -build

works as expected.

Bram

On Thu, 05 Jan 2023 08:33:37 -0800, Yubao Liu wrote

> Closed #1 as completed.
> 
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you authored the thread.Message ID: ***@***.***>

-- 
Bram van Oosterhout
